app/workers/downstream_live_worker.rb

```diff
@@ -81,4 +85,14 @@ def enqueue_dependencies
   def notify_airbrake(error, parameters)
     Airbrake.notify(error, parameters: parameters)
   end
+
+  def send_ga_event
```

Comment:
I think it would be better if this was named `send_first_time_being_published_ga_event`.